lib/utils/index.js

```diff
@@ -2558,6 +2558,12 @@ function getVueComponentDefinitionType(node) {
         const isDestructedVueComponent = isObjectArgument(parent)
         return isDestructedVueComponent ? 'defineComponent' : null
       }
+      if (callee.name === 'defineNuxtComponent') {
+        // for Nuxt 3.x
+        // defineNuxtComponent({})
+        const isDestructedVueComponent = isObjectArgument(parent)
+        return isDestructedVueComponent ? 'defineComponent' : null
```

Comment:
Shouldn't this be `defineNuxtComponent`?

```suggestion
        return isDestructedVueComponent ? 'defineNuxtComponent' : null
```